labs/9.md

```diff
@@ -0,0 +1,124 @@
+# Working with Replica Scheduling Preference 
+ReplicaSchedulingPreference provides an automated mechanism of distributing and maintaining total number of replicas for deployment or replicaset based federated workloads into federated clusters. This is based on high level user preferences given by the user. These preferences include the semantics of weighted distribution and limits (min and max) for distributing the replicas. These also include semantics to allow redistribution of replicas dynamically in case some replica pods remain unscheduled in some clusters, for example due to insufficient resources in that cluster.
+
+We will be using the reverse-words federateddeployment from Lab 9.
+
+The first schedule we will use is just a basic schedule to deploy 9 replicas across the three clusters.
+~~~sh
+cd labs/scheduling-yaml
```

Comment:
To maintain consistency with the rest of the labs, this should be changed to:

cd ~/federation-dev/labs/scheduling-yaml